docs/pages/protocol/security.mdx

```diff
@@ -87,3 +95,7 @@ Here is a summary of individual cryptographic tools used to collectively ensure
 - [Ed25519](https://ed25519.cr.yp.to/ed25519-20110926.pdf)
 
     Used for digital signatures to provide secure, high-performance signing and verification of messages. The ciphersuite is Ed25519.
+
+- [XWING KEM](https://github.com/cryspen/xwing)
```

Comment:
I would also include a reference to [ML-KEM](https://csrc.nist.gov/pubs/fips/203/final), which is the post-quantum half of XWING and an established standard.